src/Network-UUID/UUID.class.st

```diff
@@ -142,36 +135,119 @@ UUID >> asString36 [
 
 	| num |
 	num := 0.
-	1 to: self size do: [ :i | num := num + ((256 raisedTo: i - 1) * (self at: i)) ].
+	1 to: uuidData size do: [ :i | num := num + ((256 raisedTo: i - 1) * (uuidData at: i)) ].
 	^ (num printStringBase: 36) asLowercase
 ]
 
+{ #category : #accessing }
+UUID >> at: aNumber [
+	
+	"Should be deprecated, but is still used by iceberg."
+	"self
+		deprecated:
+		'The UUIDs are no longer a subclass of ByteArray, and should not be directly modified.'."
+	^ uuidData at: aNumber
+]
+
+{ #category : #accessing }
+UUID >> at: aNumber put: aByte [
```

Comment:
This Method should be deprecated, but is currently used by Iceberg. Will change it after Iceberg is updated.